src/stan/mcmc/hmc/xhmc/adapt_unit_e_xhmc.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef STAN_MCMC_HMC_xhmc_ADAPT_UNIT_E_XHMC_HPP
```

Comment:
Technically it would be slower due to the additional multiplication by 1's everywhere.